packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -36,6 +37,17 @@ function onCompositionEnd(e: Event) {
   }
 }
 
+const pad = (num: number, cover: number) => {
+  return String('0'.repeat(cover) + num).slice(-cover)
+}
+
+const toFormattedDateString = (value: Date) => {
+  if (!isDate(value)) return value
+  const year = pad(value.getFullYear(), 4)
+  const month = pad(value.getMonth() + 1, 2)
+  const date = pad(value.getDate(), 2)
```

Comment:
Shouldn't these be using their UTC equivalents?

As far as I can tell, the browser creates a Date object with the time set to midnight UTC for `valueAsDate`. But for people in timezones with negative offsets (e.g. the Americas), that corresponds to a local time on the previous day.

In my testing, I set my timezone to `-0500` and then picked a date using the dropdown. The input then jumps to the previous day.

Reply:
thank your review , 
what you found out is right ,  i edit my code , here ⬇️
```ts
  const year = pad(value.getUTCFullYear(), 4)
  const month = pad(value.getUTCMonth() + 1, 2)
  const date = pad(value.getUTCDate(), 2)
```